packages/neos-ui/src/Containers/EditModePanel/Panel/slick-styles.vanilla-css

```diff
@@ -38,6 +38,7 @@
     top: 0;
     left: 0;
     display: block;
+    width: 5000px;
```

Comment:
Hey, thanks for the fix! Magic number doesn't look good. Maybe something like `right: 0` could help here? Can't test right now.

Reply:
Unfortunately, that doesn't help. Slick calculates a width via Javascript and writes it there inline.
I think because the slider is not visible at the beginning, Slick can't calculate a width.

Reply:
Do the container has any width? if so could we use the width of the container?

Reply:
No, slick-track must be as big as all slides in the slider. Slick actually calculates this himself, which unfortunately doesn't seem to work in this case.

Reply:
Could you please at least write a comment into the code why we need this? 
This would be helpful in the future. Maybe also link to the PR in the comment.

Reply:
I added a comment.

Reply:
Thanks!